cli/ops/runtime.rs

```diff
@@ -78,3 +79,67 @@ fn op_metrics(
     "bytesReceived": m.bytes_received
   })))
 }
+
+fn ppid() -> Value {
+  #[cfg(unix)]
+  {
+    use std::os::unix::process::parent_id;
+    serde_json::to_value(parent_id()).unwrap()
+  }
+  #[cfg(not(unix))]
+  {
+    ppid_win()
+  }
```

Comment:
Shouldn't the config be the other way around? e.g. use Unix by default, have Windows be the special case.

Reply:
I think using Windows by default is better :thinking:

```rust
fn ppid() -> Value {
  #[cfg(windows)]
  {
    ppid_win()
  }
  #[cfg(not(windows))]
  {
    use std::os::unix::process::parent_id;
    serde_json::to_value(parent_id()).unwrap()
  }
}
```

Reply:
@uki00a Yep, that's exactly what I meant. :)

Reply:
Thanks @MarkTiedemann! I've made the changes you pointed out at https://github.com/denoland/deno/pull/6539/commits/2ccdd7b546052450cf9dd1044df0ea51dc553db2.